MobileWallet/Screens/RestoreWalletFromSeeds/RestoreWalletFromSeedsModel.swift

```diff
@@ -0,0 +1,130 @@
+//  RestoreWalletFromSeedsModel.swift
+
+/*
+	Package MobileWallet
+	Created by Adrian Truszczynski on 12/07/2021
+	Using Swift 5.0
+	Running on macOS 12.0
+
+	Copyright 2019 The Tari Project
+
+	Redistribution and use in source and binary forms, with or
+	without modification, are permitted provided that the
+	following conditions are met:
+
+	1. Redistributions of source code must retain the above copyright notice,
+	this list of conditions and the following disclaimer.
+
+	2. Redistributions in binary form must reproduce the above
+	copyright notice, this list of conditions and the following disclaimer in the
+	documentation and/or other materials provided with the distribution.
+
+	3. Neither the name of the copyright holder nor the names of
+	its contributors may be used to endorse or promote products
+	derived from this software without specific prior written permission.
+
+	THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND
+	CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+	INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+	OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+	DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+	CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+	SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
+	NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+	LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+	HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+	CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+	OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+	SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+import Combine
+
+final class RestoreWalletFromSeedsModel {
+
+    final class ViewModel {
+        @Published var error: SimpleErrorModel?
+        @Published var isConfimationEnabled: Bool = false
+        @Published var isEmptyWalletCreated: Bool = false
+    }
+
+    // MARK: - Properties
+
+    @Published var seedWords: [String] = []
```

Comment:
These words should be scrubbed/zero'ed out after use and should be in memory in clear text for the shortest time possible. I don't know swift well,so I don't know what's possible